x-pack/plugins/security_solution/server/utils/with_security_span.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { SpanOptions, withSpan } from '@kbn/apm-utils';
+import agent from 'elastic-apm-node';
+
+type Span = Exclude<typeof agent.currentSpan, undefined | null>;
+
+export const withSecuritySpan = <T>(
+  optionsOrName: SpanOptions | string,
+  cb: (span?: Span) => Promise<T>
+) =>
+  withSpan<T>(
+    {
+      type: 'security solution',
```

Comment:
This isn't a special value or anything is it? Just for ID purposes? Should we use `APP_ID` or another const from https://github.com/elastic/kibana/blob/5d44d79c2ba6bd0e15279b1cdf3fc5046f8e8c9d/x-pack/plugins/security_solution/common/constants.ts#L19

Reply:
It's just an ID that helps us distinguish our spans from everything else, and it could hold any value. So I don't mind changing it to `APP_ID `.